playbooks/domon-https.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create objects in Icinga
+  hosts: localhost
+  gather_facts: false
+  vars:
+    object_name: domon-https
```

Comment:
This object is only available on our instances. Maybe we should use a variable for this too.

Reply:
All required service templates are included in the playbooks, now